zproject/backends.py

```diff
@@ -1328,6 +1328,8 @@ class SocialAuthMixin(ZulipAuthMixin, ExternalAuthMethod):
     # it should be False.
     full_name_validated = False
 
+    standard_relay_params = settings.SOCIAL_AUTH_FIELDS_STORED_IN_SESSION + ['next']
```

Comment:
@mateuszmandera should we just be moving `next` into the setting and using the setting here?

Reply:
We have our own plumbing of the `next` parameter throughout the auth process, so adding it to `SOCIAL_AUTH_FIELDS_STORED_IN_SESSION` seems error prone. Would be nice to clean it up if possible though, as a follow-up perhaps